packages/server-core/src/user/avatar/avatar.ts

```diff
@@ -54,4 +57,23 @@ export default (app: Application): void => {
 
   const service = app.service(avatarPath)
   service.hooks(hooks)
+
+  service.publish('patched', async (data: AvatarType, context) => {
+    try {
+      const { params } = context
+      let targetIds = [params.user?.id]
+      const usersWithAvatar = (
+        (await app.service(userPath).find({
```

Comment:
We can use `_find` here inorder to avoid running resolvers as all we need is user ids from this API call.

Reply:
Done